tutorials/scripting/gdscript/gdscript_exports.rst

```diff
@@ -178,6 +178,16 @@ when editing.
 
     @export_exp_easing var transition_speed
 
+Export with Suffix Hint
+------
+
+Display a unit hint suffix for exported variables.
+
+::
+
+    @export_custom(PROPERTY_HINT_NONE, "suffix:m/s\u00b2") var gravity
```

Comment:
```suggestion
    @export_custom(PROPERTY_HINT_NONE, "suffix:m/s\u00b2") var gravity: float
```

The original line you had compiles, but is not editable. 
Quick testing:
```
@export_custom(PROPERTY_HINT_NONE, "suffix:m/s\u00b2") var invalid
@export_custom(PROPERTY_HINT_NONE, "suffix:m/s\u00b2") var valid_0 : float
@export_custom(PROPERTY_HINT_NONE, "suffix:m/s\u00b2") var valid_1 = 0.0
@export_custom(PROPERTY_HINT_NONE, "suffix:m/s\u00b2") var valid_2 := 0.0

@export_custom(PROPERTY_HINT_NONE, "suffix:m/s\u00b2") var invalid_int
@export_custom(PROPERTY_HINT_NONE, "suffix:m/s\u00b2") var valid_int_0 : int
@export_custom(PROPERTY_HINT_NONE, "suffix:m/s\u00b2") var valid_int_1 = 0
@export_custom(PROPERTY_HINT_NONE, "suffix:m/s\u00b2") var valid_int_2 := 0
```
![Godot_v4 3-stable_mono_win64_RIvv12JRU7](https://github.com/user-attachments/assets/af703884-b66b-4a4d-97a6-002322db2160)